gpkit/constraints/model.py

```diff
@@ -133,7 +134,7 @@ def _add_modelname_tovars(self, name, num):
         with SignomialsEnabled():  # since we're just substituting varkeys.
             self.subinplace(add_model_subs)
 
-    # pylint: disable=too-many-locals
+    # pylint: disable=too-many-locals,too-many-branches,too-many-statements
```

Comment:
Oh, I thought when it was on the line above a function dec it only applied to that function...